src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourtCase.kt

```diff
@@ -106,7 +106,7 @@ class CourtCase(
   val lidsCombinedCaseId: Int? = null,
 
   @OneToMany(mappedBy = "courtCase", cascade = [CascadeType.ALL], orphanRemoval = true)
-  val courtEvents: MutableList<CourtEvent> = mutableListOf(),
+  var courtEvents: MutableList<CourtEvent> = mutableListOf(),
```

Comment:
interesting, would have thought it would remain as a val and you just the list contents

Reply:
I see, makes my sense to assign on a POST